numpy/ma/core.py

```diff
@@ -193,7 +193,7 @@ class MaskError(MAError):
 
     if scalar_dtype.kind in "Mm":
         info = np.iinfo(np.int64)
-        min_val, max_val = info.min, info.max
+        min_val, max_val = info.min + 1, info.max
```

Comment:
Since we are hard-coding this value.  It would be nice to cover these "interesting values" in the test also.  (Maybe including `NaT` itself.)

Reply:
I have add test to `NaT` in the last PR.